sup3r/qa/utilities.py

```diff
@@ -139,27 +169,35 @@ def wavenumber_spectrum(var, k_range=None, axis=0):
     return k[:n_steps], E_k
 
 
-def tke_series(u, v):
-    """Longitudinal Turbulent Kinetic Energy Spectrum time series. Gives the
-    mean tke spectrum over time.
+def direct_dist(var, bins=40, range=None, diff_max=None, scale=1):
+    """Returns the direct distribution for the given variable.
 
     Parameters
     ----------
-    u: ndarray
-        (lat, lon, time)
-        U component of wind
-    v : ndarray
-        (lat, lon, time)
-        V component of wind
+    var: ndarray
+        (lat, lon, temporal)
+    bins : int
+        Number of bins for the direct pdf.
+    range : tuple | None
+        Optional min/max range for the direct pdf.
+    diff_max : float
+        Max value to keep for given variable
 
     Returns
     -------
     ndarray
-        1D array of mean tke amplitudes over time
+        var at bin centers
+    ndarray
+        Normalized var value counts
+    float
+        Normalization factor
     """
-
-    return [np.mean(tke_wavenumber_spectrum(u[..., t], v[..., t]))
-            for t in range(u.shape[-1])]
+    diffs = var / scale
+    diff_max = diff_max or np.percentile(np.abs(diffs), 99.9)
```

Comment:
This is a lot of implicit logic. I think the percentile should be a kwarg and described. I also think this operation should be optional by default. It might make sense for wind but not for any other dataset. Also shouldn't the range arg take care of this? Finally, add docstring for `scale` param

Reply:
Yeah range could accomplish the same thing, so could diff_max, but if I don't know the range I want this is a nice default imo. I'd rather not have to manually specify the range to exclude weird outliers.

Reply:
okayyy at least move it to a kwarg and describe it in the docstrings so its not implicit

Reply:
haha okay sounds good.